tests/models/bert/test_tokenization_bert_tf.py

```diff
@@ -41,7 +41,10 @@ def setUp(self):
         super().setUp()
 
         self.tokenizers = [BertTokenizer.from_pretrained(checkpoint) for checkpoint in TOKENIZER_CHECKPOINTS]
-        self.tf_tokenizers = [TFBertTokenizer.from_pretrained(checkpoint) for checkpoint in TOKENIZER_CHECKPOINTS]
+        self.tf_tokenizers = [TFBertTokenizer.from_pretrained(checkpoint) for checkpoint in TOKENIZER_CHECKPOINTS] + [
```

Comment:
Because we are doubling the size of the `self.tf_tokenizers`, the list above (`self.tokenizers`) will also have to be repeated twice :)

This is because both lists are iterated on with a `zip`, and `zip` ends as soon as one iterator ends. In other words, the tests are not being executed for the new tokenizers with `use_fast_bert_tokenizer=False`.

Example of `zip` ending as soon as one iterator ends:
```python
a = [1, 2]
b = [5, 6, 7]
for c, d in zip(a, b):
  print(c, d)
```

Reply:
Thank you for catching that, just fixed it.